app/models/container_template.rb

```diff
@@ -0,0 +1,15 @@
+class ContainerTemplate < ApplicationRecord
+  include CustomAttributeMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
```

Comment:
IIRC this is used across all entities for faster lookup by provider (please check other entities).